packages/strapi-admin/admin/src/containers/Roles/ListPage/RoleDescription.js

```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+import { Text } from '@buffetjs/core';
+
+const RoleDescription = styled(Text)`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  max-width: 25rem;
+  white-space: nowrap;
+`;
```

Comment:
In the future, wi will need to improve the text component to handle the ellipsis mode.

Reply:
It already handles it according to this [file](https://github.com/strapi/buffet/blob/65f190a565e8220c59210bf1170230f04034d6e5/packages/buffetjs-core/src/components/Text/index.js#L12-L18) so you can use the `ellipsis` props

Reply:
Awesome thank you I didn't see this.